### ISSUE TITLE
Comment threads

### ISSUE THREAD

**Issue by SchalkCloete** [MyResonance/tickets#25]
The ability to directly reply to comments will make the experience more engaging.

**Comment by blamarque** [comment]
Hello there,

Do you mean a sub comment of a comment ?

**Comment by SchalkCloete** [comment]
Yes, each comment can have a little "Reply" button that allows people to reply precisely to that comment so that the reply stands below the right comment and the right person is automatically notified. Such sub comments often have an indent (shifted a little to the right) so that it's easy to see they are a reply to the comment above them. It's probably smart to allow only 1 or 2 layers of such sub-comments to avoid the comment tree from ever becoming too complicated.

**Comment by SchalkCloete** [comment]
It should also be possible to "like" someone else's comment to acknowledge that you've seen it when you don't have anything concrete to reply. On MS Teams, they have a nice feature that you can hover over the comment and get options like a thumbs-up, heart, laughing face, or frowning face that you can click to quickly send back an acknowledgment.
